tnt-persistence/src/main/java/org/dcsa/tnt/persistence/repository/specification/EventCacheSpecification.java

```diff
@@ -55,6 +68,9 @@ public static Specification<EventCache> withFilters(final EventCacheFilters filt
         predicates.add(root.get(EventCache_.EVENT_TYPE).in(filters.eventType));
       }
 
+      /*
+      TODO rewrite/update these filters for 3.0 spec filters
```

Comment:
Created https://dcsa.atlassian.net/browse/DDT-1265 to update filters and integration tests